model/registry/user.yaml

```diff
@@ -0,0 +1,59 @@
+groups:
+  - id: registry.user
+    prefix: user
+    type: attribute_group
+    brief: "Describes information about the user that is relevant to the event."
+    attributes:
+      - id: domain
+        type: string
+        stability: experimental
+        brief: >
+          Name of the directory the user is a member of. For example, an LDAP or Active Directory domain name.
+        examples: ['internal.example.com']
+      - id: email
+        type: string
+        stability: experimental
+        brief: >
+          User email address.
+        examples: ['a.einstein@example.com']
+      - id: full_name
+        type: string
+        stability: experimental
+        brief: >
+          User's full name
+        examples: ['Albert Einstein']
+      - id: hash
+        type: string
+        stability: experimental
+        brief: >
+          Unique user hash to correlate information for a user in anonymized form.
+        note: >
+          Useful if `user.id` or `user.name` contain confidential information and cannot be used.
+        examples: ['364fc68eaf4c8acec74a4e52d7d1feaa']
+      - id: id
+        type: string
+        stability: experimental
+        brief: >
+          Unique identifier of the user.
+        examples: ['S-1-5-21-202424912787-2692429404-2351956786-1000']
+      - id: name
+        type: string
+        stability: experimental
+        brief: >
+          Short name or login/username of the user.
+        examples: ['a.einstein']
+      - id: roles
+        type: string[]
+        stability: experimental
+        brief: >
+          Array of user roles at the time of the event.
+        examples: ["admin", "reporting_user"]
+      - id: scope
```

Comment:
we should try to find more specific term for it.

From non-security context `user.scope` could be absolutely anything. I'd be surprised to find `Scopes or granted authorities the client currently possesses extracted from token or application security context.`  there.

If we're adding a identity/auth/security user, let's pick a corresponding namespace for it.

Reply:
This field is coming from original [enduser](https://github.com/open-telemetry/semantic-conventions/blob/main/model/registry/enduser.yaml#L21) namespace for backward compatibility

Reply:
Since the `enduser` is being deprecated, we don't need to stay backward compatible with it and can do a better job picking attribute names.

Reply:
let's remove it then